test/metabase/upload_test.clj

```diff
@@ -20,13 +20,18 @@
 
 (set! *warn-on-reflection* true)
 
-(def ^:private bool-type      :metabase.upload/boolean)
-(def ^:private int-type       :metabase.upload/int)
-(def ^:private float-type     :metabase.upload/float)
-(def ^:private vchar-type     :metabase.upload/varchar_255)
-(def ^:private date-type      :metabase.upload/date)
-(def ^:private datetime-type  :metabase.upload/datetime)
-(def ^:private text-type      :metabase.upload/text)
+(def ^:private bool-type        :metabase.upload/boolean)
+(def ^:private int-type         :metabase.upload/int)
+(def ^:private float-type       :metabase.upload/float)
+(def ^:private vchar-type       :metabase.upload/varchar_255)
+(def ^:private date-type        :metabase.upload/date)
+(def ^:private datetime-type    :metabase.upload/datetime)
+(def ^:private text-type        :metabase.upload/text)
+(def ^:private pk-type          :metabase.upload/pk)
+(def ^:private pk-schema [:map
+                          [:type [:= pk-type]]
+                          [:opts  [:vector keyword?]]
+                          [:exclude-in-insert? {:optional true} boolean?]])
```

Comment:
If you're just going to test that PKs adhere to a schema, I think it's better to add the schema to the code with `mu/defn` and skip using `malli=`. That way you get the benefits of documentation that comes with malli, but in the code. And the tests can stick to testing values, not types.